test/libyul/evmCodeTransform/stackReuse/function_argument_reuse_without_retparams.yul

```diff
@@ -6,7 +6,7 @@
 }
 // ====
 // stackOptimization: true
-// EVMVersion: >=shanghai
+// EVMVersion: =current
```

Comment:
Yeah, fair enough, could be said for most of these tests though (although not necessarily with shanghai but with other versions). It was my understanding that we said the code transform tests should really only be tested against the latest as there are no guarantees beyond that :) i'll change it back in any case.

Reply:
Run it.
Optimizer run boost 
Stack

Reply:
I don't think it's really "should". The transform is supposed to work with older versions too and ideally all of them should covered. It's just that in most cases covering more than a single version is a hassle because the output is different and you'd need multiple tests.

But if we have a test that was prepared with multiple versions in mind (assuming it was not just a mistake), we should let it be :)

Reply:
Yeah, fair!